File: DAY48/a2.py
class Account:
    def __init__(self,owner,balance):
        self.owner = owner
        self.__balance = balance

    def deposit(self,amount):
        if amount > 0:
            self.__balance += amount
            print(f"Added {amount} to the balance")
        else:
            print("Deposit amount must be positive")

    
    def get_balance(self):
        return self.__balance


acc = Account("John",1000)
print(acc.get_balance())
acc.deposit(500)
print(acc.get_balance())

File: DAY48/a2.java
public class Account{
    public String owner;
    public double balance;


    public Account(String onwer,double balance){
        this.owner = owner;
        this.balance = balance;
    }

    public void deposit(double amount){
        if(amount > 0){
            this.balance += amount;
        }
    }

    public double get_balance(){
        return this.balance;
    }

    public static void main(String[] args){
        Account acc = new Account("John",1000);
        System.out.println(acc.get_balance());
        acc.deposit(500);
        System.out.println(acc.get_balance());
    }

}